scripts/bozzcoin.js

```diff
@@ -0,0 +1,203 @@
+// Description:
+//   Tracks bozzcoins earned through exercise
+//
+// Commands: Hubot must be specifically @-mentioned.
+//   bozzcoins? - reports how many bozzcoins are in the account.
+//   did x (pullups|pushups|situps|squats|lunges|racket steps)
+//      - adds bozzcoins to the account depending on the exercise type.
+//   ran x km - adds 100 bozzcoins per km.
+//   (prata|starbucks|macs) day - subtracts bozzcoins depending on the cheat type.
+
+module.exports = (robot) => {
```

Comment:
Hmm I thought arrow functions are ES6, not plain JS?

Reply:
Yep it's ES6. Should we not use ES6?

Reply:
We should, if hubot already supports it

Reply:
Yep it does. At least locally.